src/client/Client.ts

```diff
@@ -44,64 +48,44 @@ class Bot extends Client {
     private async loadCommands() {
         this.logger.info(`Loading commands...`);
 
-        const commandFiles: string[] = fs
-            .readdirSync(path.resolve(__dirname, '../commands'))
-            .filter((file) => file.endsWith('.js'));
-
-        for (const file of commandFiles) {
-            const command: Command = require(`../commands/${file}`);
+        Object.values(commands).forEach(command => {
             this.commands.set(command.data.name, command);
             this.logger.success(`Command ${command.data.name} loaded.`);
-        }
+        });
 
         this.logger.success(`Commands loaded.`);
     }
 
     private async loadEvents() {
         this.logger.info(`Loading events...`);
 
-        const eventFiles = fs
-            .readdirSync(path.resolve(__dirname, '../events'))
-            .filter((file) => file.endsWith('.js'));
-
-        for (const file of eventFiles) {
-            const event = require(`../events/${file}`);
-            if (event.once) {
-                this.once(event.name, (...args) => event.execute(...args));
-            } else {
-                this.on(event.name, (...args) => event.execute(...args));
-            }
-            this.logger.success(`Listening to ${event.name} event.`);
-        }
+	Object.values(events).forEach(event => {
+	    if (event.once) {
+		this.once(event.name, (...args) => event.execute(...args));
+	    } else {
+		this.on(event.name, (...args) => event.execute(...args));
+	    }
+	    this.logger.success(`Listening to ${event.name} event.`);
+	});
 
         this.logger.success(`Events loaded.`);
     }
 
     private async loadButtons() {
         this.logger.info(`Loading buttons...`);
-
-        const buttonFiles: string[] = fs
-            .readdirSync(path.resolve(__dirname, '../components/buttons'))
-            .filter((file) => file.endsWith('.js'));
-
-        for (const file of buttonFiles) {
-            const button: Button = require(`../components/buttons/${file}`);
+	
+	Object.values(buttons).forEach(button => {
             this.buttons.set(button.data.customId!, button);
             this.logger.success(`Button ${button.data.customId} loaded.`);
-        }
+	}
```

Comment:
```suggestion
	});
```

Puede ser que haya faltado cerrar la función?

Reply:
Sip, bien visto! Ahora lo arreglo